README.md

```diff
@@ -6,6 +6,7 @@
 [![open issues](https://img.shields.io/github/issues-raw/dbartholomae/jsx-readme.svg)](https://github.com/dbartholomae/jsx-readme/issues)
 [![dependency status](https://david-dm.org/dbartholomae/jsx-readme.svg?theme=shields.io)](https://david-dm.org/dbartholomae/jsx-readme)
 [![devDependency status](https://david-dm.org/dbartholomae/jsx-readme/dev-status.svg)](https://david-dm.org/dbartholomae/jsx-readme?type=dev)
+[![language](https://img.shields.io/github/languages/top/dbartholomae/jsx-readme)](https://github.com/dbartholomae/jsx-readme)
```

Comment:
Instead of `language` it should be `GitHub top language`. For the link I like the choice of linking to the repo :)

Reply:
Hi, isn't the link already linking to the Github repository ?

Reply:
Yes, that's what I meant: It was not specified in the ticket, but I like how it is done in the PR.